src/safeposix/syscalls/net_constants.rs

```diff
@@ -237,25 +237,34 @@ pub const SHUT_RDWR: i32 = 2;
 
 ////////////////////// setsockopt / getsockopt...
 pub const SOL_SOCKET: i32 = 1;
```

Comment:
are SOL_SOCKET and SO_DEBUG both supposed to be 1?

Reply:
Yep
https://elixir.bootlin.com/linux/latest/source/include/uapi/asm-generic/socket.h#L9